tree/dataframe/inc/ROOT/RDF/InterfaceUtils.hxx

```diff
@@ -92,6 +92,7 @@ struct Histo3D{};
 struct HistoND{};
 struct HistoNSparseD{};
 struct Hist{};
+struct HistWithWeight{};
```

Comment:
Having a second tag felt like the easiest solution, together with just a template argument `bool WithWeight = false` for `RHistFillHelper`. Other approaches are certainly possible...